cropper/src/main/java/com/canhub/cropper/CropImageContractOptions.kt

```diff
@@ -500,6 +501,32 @@ data class CropImageContractOptions @JvmOverloads constructor(
         cropImageOptions.showCropOverlay = !skipEditing
         return this
     }
+
+    /**
+     * Shows an intent chooser instead of the alert dialog when choosing an image source.
+     * *Default: false*
+     */
+    fun setShowIntentChooser(showIntentChooser: Boolean) = cropImageOptions.apply {
+        this.showIntentChooser = showIntentChooser
+    }
+
+    /**
+     * Sets a custom title for the intent chooser
+     */
+    fun setIntentChooserTitle(intentChooserTitle: String) = cropImageOptions.apply {
+        this.intentChooserTitle = intentChooserTitle
+    }
+
+    /**
+     * This takes the given app package list (list of app package names)
+     * and displays them first among the list of apps available
+     *
+     * @param priorityAppPackages accepts a list of strings of app package names
+     * Apps are displayed in the order you pass them if they are available on your device
+     */
+    fun setIntentChooserPriorityList(priorityAppPackages: List<String>) = cropImageOptions.apply {
+        this.intentChooserPriorityList = priorityAppPackages
+    }
```

Comment:
I've handled that in the `CropImageIntentChooser` class. Either way if the list is passed empty it just won't re-arrange the existing apps.

Reply:
This list is basically used for sorting the list of displayed apps in the intent chooser. I've handled empty lists passed by the user in `CropImageIntentChooser` class.

Either way if it's empty it will sort by the default list I've made that contains common apps like Google Photos, Samsung Gallery etc.

Reply:
Let's add this information to the documentation /\, so future users can know

Reply:
Done